### PR TITLE
New version

### PR DESCRIPTION
**Pull request by bhazelton**
## Description
New minor version with a few docs fixes.

## Motivation and Context

There were some problems with recent merges of the changelog that needed to be fixed (changes were noted under an old version rather than under unreleased).

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the [contribution guide](.github/CONTRIBUTING.md).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly.
- [ ] My change introduces new functionality that should be highlighted in the tutorial.
  - [ ] I have updated the tutorial accordingly.
- [ ] If this is a bug fix, it includes a new test that breaks as a result of the bug (if possible)
- [ ] If this is a breaking change, it includes backwards compatibility and deprecation warnings (if possible)
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests pass in both python 2 and python 3.
- [x] My change requires an update to the [CHANGELOG](CHANGELOG.md).
  - [x] I have updated the [CHANGELOG](CHANGELOG.md) accordingly.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,24 +3,28 @@ All notable changes to this project will be documented in this file.
...
-- `freq_interpolation_kind` added as an attribute to `UVBeam`
-- `tol` added as keyword argument to `UVBeam._interp_freq` which allows for a fast return of `data_array` slice if nearest-neighbor frequencies are all within the distance tolerance.
-- `polarizations` added as keyword argument to `UVBeam` interpolation methods.
 - `keep_all_metadata` keyword for optionally discarding unused metadata when performing a select operation.
 
 ### Changed
-- `UVBeam._interp_freq` returns both `interp_data` and `interp_bandpass`, instead of just the former.
 - Extends `run_acceptability_check` for UVH5 metadata in `check_header` function.
 
 ### Fixed
```
Comment (plaplant) [inline CHANGELOG.md]: One thing I noticed: version 1.3.4 has 2 "Added" sections. We should merge those into a single one.

Comment (bhazelton) [reply CHANGELOG.md]: good catch, thanks!

Comment (plaplant) [inline CHANGELOG.md]: Also, the years for versions 1.2 and 1.2.1 should be 2017, no? (Not sure why I'm only realizing these problems now...)

Comment (bhazelton) [reply CHANGELOG.md]: ack! Thank you, I didn't see those either.

**Pull request merged**